src/OpenSage.Game.Tests/Data/Ini/IniFileTests.cs

```diff
@@ -25,38 +26,161 @@ public void CanReadIniFiles()
                 GameDefinition.FromGame(SageGame.CncGenerals),
                 GameDefinition.FromGame(SageGame.CncGeneralsZeroHour),
                 GameDefinition.FromGame(SageGame.Bfme),
-                //GameDefinition.FromGame(SageGame.Bfme2),
+                GameDefinition.FromGame(SageGame.Bfme2),
+                GameDefinition.FromGame(SageGame.Bfme2Rotwk),
             };
 
             foreach(var gameDefinition in gameDefinitions)
             {
-                InstalledFilesTestData.ReadFiles(".ini", _output, new List<IGameDefinition> { gameDefinition }, entry =>
+                var rootDirectories = InstallationLocators.FindAllInstallations(gameDefinition)
+                .Select(i => i.Path)
+                .Where(x => Directory.Exists(x))
```

Comment:
I believe both installation locators already check whether the path exists, so this check isn't necessary.

Reply:
removed